extensions/webjars-locator/deployment/pom.xml

```diff
@@ -56,11 +56,10 @@
           <version>2.24.0</version>
           <scope>test</scope>
         </dependency>
-        
-        <!-- DevMode has no setForcedDependencies, so need to add resteasy extension here -->
         <dependency>
           <groupId>io.quarkus</groupId>
-          <artifactId>quarkus-resteasy</artifactId>
+          <artifactId>quarkus-resteasy-deployment</artifactId>
+          <scope>test</scope>
```

Comment:
This one was only used for the dev mode test but didn't have the test scope. Also let's use it for all the tests and avoid using `setForcedDependencies` with a hardcoded version.